lib/bundles/inspec-compliance/target.rb

```diff
@@ -55,6 +55,9 @@ def self.resolve(target) # rubocop:disable PerceivedComplexity, Metrics/Cyclomat
         end
         profile_fetch_url = Compliance::API.target_url(config, profile)
       end
+      # We need to pass the token to the fetcher
+      config['token'] = Compliance::API.get_token(config)
```

Comment:
Good idea. In general we want to avoid having the configuration in our API. Right now, storing it on disk is the only way to share it with the fetcher, since they run independently.